### DISCUSSION TITLE
Day 10

### DISCUSSION THREAD

**Discussion by CarolineMorton** [CarolineMorton/AdventOfCode_WIR#13] [Ideas]
🎄🎄 Day 10 🎄🎄

Like many of you, i am slightly behind but will come back to this. I am making this so we can share solutions

**Comment by AR2202** [comment]
Found [day 10](https://github.com/AR2202/AOC2024/blob/main/src/Day10.hs) relatively easy, accidentally solved part 2 on the way to solving part 1!

**Comment by Sowmiya07** [comment]
[Here](https://github.com/Sowmiya07/AdventOfCode_WIR/blob/main/day_10/src/main.rs) is my Day 10 Part 1 solution. Yet to finish part 2. I will update the thread once complete.
